index.js

```diff
@@ -793,7 +802,7 @@ class Batch {
       }
     }
 
-    return this._append(root, seq, key, value)
+    return this._append(root, seq, key, enc(encoding.value, newNode.value))
```

Comment:
move the encoded value to `const value = enc(....)` before this line

Reply:
`value` exists as arg so I named it `valueEncoded`